task_templates/README.md

```diff
@@ -1,50 +1,70 @@
 # Templates for Custom Tasks
+
 In this directory are three folders - estimators, transforms, and pipelines
 
 * estimators - meant to be used as the prediction step in a blueprint
 * transforms - meant to be used as a preprocessing step
-* pipelines - meant to be added as a single task blueprint, these have some preprocessing and an estimator built in
+* pipelines - meant to be added as a single task blueprint, these have some preprocessing and an
+  estimator built in
 
-Our estimator examples are also different than the pipelines examples because, we decided to implement the score
-hooks for all the estimator templates, and for the pipelines templates we did not. You only don't have to use a score
-hook if your model inherits from sklearn, pytorch, keras, or xgboost and doesn't need any additional functionality
+Our estimator examples are also different than the pipelines examples because, we decided to
+implement the score hooks for all the estimator templates, and for the pipelines templates we did
+not. You only don't have to use a score hook if your model inherits from sklearn, pytorch, keras, or
+xgboost and doesn't need any additional functionality
 
 Below is an enumeration of the templates we provide
 
 ## Transforms
-* `custom_class_python` - Shows how to define your own transformation class without relying on external libraries
+
+* `custom_class_python` - Shows how to define your own transformation class without relying on
+  external libraries
 * `python_missing_values` - Uses builtin pandas functionality to impute missing values
```

Comment:
Let's make python_missing_values the first example. it's the simplest example we have for transforms, would be good to put it first.
Shall we also outline that it's the simplest example? E.g. we can describe it as "Simple transform implementing missing values imputation"